rust/lance/src/dataset/write/merge_insert/exec/mod.rs

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+mod write;
```

Comment:
Minor nit: Why using the `mod.rs` pattern here?  We typically use the other pattern everywhere else it seems?  Not really important.

Reply:
I've changed to be consistent.